### PR TITLE
compress_weights telemetry update

### PR DESCRIPTION
**Pull request by KodiaqQ**
### Changes

- Added telemetry for the `nncf.compress_weights` backend-specific `impl`;
- Added missed wrapper for the `nncf.qantize` PyTorch `impl`;
- Fixed reported event for `nncf.quantize_with_accuracy_control` OpenVINO `impl`.
- Added missed `app_name`, `app_version` fields for each event.

### Reason for changes

- Telemetry update.

### Related tickets

- 154833

### Tests

- TBD

### CODE REVIEW

**Review by AlexKoff88 — APPROVED** [review]

File: nncf/openvino/quantization/quantize_model.py
Comment (MaximProshin) [inline nncf/openvino/quantization/quantize_model.py:375]: I would extend it by 'all_layers'. cc @AlexKoff88

Comment (ljaljushkin) [reply nncf/openvino/quantization/quantize_model.py:375]: What about `ratio` and `group_size`?

**Review by alexsu52 — APPROVED** [review]

File: nncf/openvino/quantization/quantize_model.py
Comment (MaximProshin) [reply nncf/openvino/quantization/quantize_model.py:375]: > What about `ratio` and `group_size`?

How do you plan to use that information?

**Comment by MaximProshin** [comment]
Every new data in telemetry requires a legal approval and we can't merge the PR until then. I will try to force it so it doesn't take a lot of time. 
@alexsu52 @AlexKoff88 fyi

File: nncf/openvino/quantization/quantize_model.py
Comment (ljaljushkin) [reply nncf/openvino/quantization/quantize_model.py:375]: It might show how much the ratio and group size have been reduced in order to obtain an accurate model, as an example.

Comment (MaximProshin) [reply nncf/openvino/quantization/quantize_model.py:375]: Though for gs it's still ok as I think most of users will have 64 or 128

Comment (ljaljushkin) [reply nncf/openvino/quantization/quantize_model.py:375]: Ratio would show how often mixed precision is used, and how often customers choose full int4 model.
Often usage of mixed precision with small ratio and low group size might hint the need for more advanced technics to recover accuracy, as vector quantization, fine-tuning ...

Comment (MaximProshin) [reply nncf/openvino/quantization/quantize_model.py:375]: Agreed to keep only mode and algo.

**Review by ljaljushkin — APPROVED** [review]

**Review by MaximProshin — APPROVED** [review]
Feel free to merge it

**Pull request merged**
